resources/views/home.blade.php

```diff
@@ -1,70 +1,161 @@
-                </form>
-            </div>
-        </div>
-    </div>
+	<div>
+		<div id="main" class="home view-projects">
+			<div class="overlay">
+				<div class="contents">
+					<h1 class="ui header">Welcome to Bluemarket</h1>
+					<a class="ui primary button" href="/projects"><div class="bluemarket-button">View projects</div></a>
+				</div>
+			</div>
+		</div>
+	</div>
+	<div class="padded content">
+		<div class="home info">
+			<div class="ui divider"></div>
+				<div class="ui two column padded grid">
+					<div class="column">
+						<h1 class="bluemarket-text">What is Bluemarket?</h1>
+						<p class="bluemarket-text">Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.</p>
+					</div>
+					<div class="column image-container">
+						<img class="ui small circular image" src="https://images.unsplash.com/photo-1531496681078-27dc2277e898?ixlib=rb-1.2.1&q=80&fm=jpg&crop=entropy&cs=tinysrgb&w=200&h=200&fit=crop&ixid=eyJhcHBfaWQiOjF9">
+					</div>
+				</div>
+			</div>
+		</div>
+		<div class="content ui padded grid">
+			<div class="home info steps">
+				<h1 class="bluemarket-text">Getting started</h1>
+				<ul class="progress">
+					<li class="progress-item">
+						<p class="progress-title bluemarket-text">Step 1</p>
+						<p class="progress-info bluemarket-text">Register in the platform</p>
+					</li>
+					<li class="progress-item">
+						<p class="progress-title bluemarket-text">Step 2</p>
+						<p class="progress-info bluemarket-text">Find and submit projects</p>
+					</li>
+					<li class="progress-item">
+						<p class="progress-title bluemarket-text">Step 3</p>
+						<p class="progress-info bluemarket-text">Find and collaborate with people</p>
+					</li>
+				</ul>
+			</div>
+		</div>
+		<div class="padded content contact ui grid">
+			<div class="home info contact">
+				<h1 class="bluemarket-text">Contact us</h1>
+				<a name="contact"></a>
+				<div class="ui form error">
+					<!-- Name -->
+					<div id="nameInput" class="field input">
+						<label for="name">Name</label>
+						<input type="text" id="name" name="name" placeholder="e.g. John Doe">
+					</div>
+
+					<!-- Email -->
+					<div id="emailInput" class="field input">
+						<label for="email">Email</label>
+						<input type="text" id="email" name="email" placeholder="e.g. john@example.com">
+					</div>
+
+					<!-- Message -->
+					<div id="messageInput" class="field input">
+						<label for="message">Message</label>
+						<textarea id="message" name="message"></textarea>
+					</div>
+
+					<!-- Success message -->
+					<div id="success" class="ui positive message">
+					<div class="header">
+						Thank you for your message.
+					</div>
+					<p>We'll get back at you as soon as possible.</p>
```

Comment:
We'll get back **to** you as soon as possible. Creo que tiene otro significado *get back at someone* (take revenge on someone)

Reply:
![Alt Text](https://media.giphy.com/media/NppB385x566nm/giphy.gif)